core/syntax/syntax-docs.factor

```diff
@@ -671,16 +671,16 @@ HELP: NAN:
 HELP: GENERIC:
 { $syntax "GENERIC: word ( stack -- effect )" }
 { $values { "word" "a new word to define" } }
-{ $description "Defines a new generic word in the current vocabulary. Initially, it contains no methods, and thus will throw a " { $link no-method } " error when called." } ;
+{ $description "Defines a new generic word in the current vocabulary. The word dispatches on the topmost stack element. Initially it contains no methods, and thus will throw a " { $link no-method } " error when called." } ;
```

Comment:
What do you mean by "dispatching on the topmost stack element"?

Reply:
I mean that it takes the top element on the stack, tests its class and calls the appropriate method based on the class. You know, [method dispatch](https://en.wikipedia.org/wiki/Dynamic_dispatch).